lib/db/mysql.js

```diff
@@ -95,7 +95,8 @@ const Q_AVATAR_LIST = 'SELECT avatars.id, url, avatar_selected.avatarId '
 const Q_AVATAR_DELETE = 'DELETE FROM avatars WHERE id=?';
 
 const Q_PROVIDER_INSERT = 'INSERT INTO avatar_providers (name) VALUES (?)';
-const Q_PROVIDER_GET = 'SELECT * FROM avatar_providers WHERE name=?';
+const Q_PROVIDER_GET_NAME = 'SELECT * FROM avatar_providers WHERE name=?';
+const Q_PROVIDER_GET_ID = 'SELECT * FROM avatar_providers WHERE id=?';
```

Comment:
nit: I would suggest `Q_PROVIDER_GET_BY_NAME`/`Q_PROVIDER_GET_BY_ID` here to match the naming of the functions that use them.  The current naming could ambiguously suggest that we're "getting the name" rather than "getting by name".